src/dfmt/formatter.d

```diff
@@ -1342,7 +1399,8 @@ private:
             regenLineBreakHintsIfNecessary(index);
             immutable bool ufcsWrap = astInformation.ufcsHintLocations.canFindIndex(current.index);
             if (ufcsWrap || linebreakHints.canFind(index) || (linebreakHints.length == 0
-                    && currentLineLength + nextTokenLength() > config.max_line_length))
+                    && currentLineLength + nextTokenLength() > config.max_line_length)
+                    || onNextLine)
```

Comment:
I think this reads a bit more clearly if you put onNextLine next to the other short conditions.
```d
if (ufcsWrap || linebreakHints.canFind(index) || onNextLine
        || (linebreakHints.length == 0 && currentLineLength + nextTokenLength() > config.max_line_length))
```